OpenRA.Mods.Common/InstallUtils.cs

```diff
@@ -86,6 +86,9 @@ public static string GetMountedDisk(Func<string, bool> isValidDisk)
 		public static bool CopyFiles(string srcPath, Dictionary<string, string[]> files, string destPath,
 			bool overwrite, Action<string> onProgress, Action<string> onError)
 		{
+			if (!Directory.Exists(destPath))
```

Comment:
No need to test.
[The docs say the directory will only be created if it doesn't already exist.](https://msdn.microsoft.com/en-us/library/54a0at6s%28v=vs.110%29.aspx)